src/content/learn/rendering-lists.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: Rendering Lists
+title: Me-render List
```

Comment:
```suggestion
title: Merender List
```

Reply:
Sayangnya, *render* belum menjadi kata baku di Bahasa Indonesia, bisa dilihat di [KBBI](https://kbbi.kemdikbud.go.id/entri/render). Oleh karena itu, penulisan dengan imbuhan harus dipisah dengan menggunakan tanda hubung sesuai [PUEBI](https://puebi.js.org/tanda/hubung), poin III.E.6.

Reply:
@mazipan @samnap11 Iya ini jadi masih perlu ditetapkan mau yang mana bersama dengan @r17x @resir014 @zainfathoni dkk terutama di komunitas. Karena ada yang sesuai kaidah bahasa umum untuk penerjemahan (me-render) vs tidak kaidah tapi lebih ke sederhana (merender).

Saya pribadi selama bertahun-tahun (sejak 2010) menerjemahkan dokumen teknis, selalu mengikuti kaidah bahasa yang berlaku.

Reply:
Yuk pakai me-*render* untuk selanjutnya.
Yang keburu di merge, nanti di cari ulang, dan di bantu benahi nyusul aja

Reply:
@mazipan 😂 Akhirnya setelah sekian hari wkwk Kalo cari ulang juga nanti relatif mudah, tinggal find and replace.